Adds documentation for virtual bridge error (#183)
unikraft/docs: content/en/community/hackathons/sessions/baby-steps/content/Demos_02.-Building-and-Running-the-httpreply-Application.md

Listening on port 8123...
```

> In this step you may face an error : 
> ```

Reviewer: ```suggestion
In this step you may face the following error:

```text
```

access denied by acl file
qemu-system-ppc64: -netdev bridge,br=virbr0,id=net0: bridge helper failed
```

> For resolving this, check if `/etc/qemu/bridge.conf` exists and if it does add the line `allow virbr0` to it to allow your bridge to run.

Reviewer: ```suggestion

To fix this, check if the `/etc/qemu/bridge.conf` file exists and if it does add the line `allow virbr0` to it to allow your bridge to run.
```

Listening on port 8123...
```

In this step you may face the following error:
```text
access denied by acl file
qemu-system-ppc64: -netdev bridge,br=virbr0,id=net0: bridge helper failed
```

> To fix this, check if the `/etc/qemu/bridge.conf` file exists. If it doesn't, then create it first.and then add the line `allow virbr0` to it to allow your bridge to run.

Reviewer 2: ```suggestion
To fix this, check if the `/etc/qemu/bridge.conf` file exists.
If it doesn't, then create it first and then add the line `allow virbr0` to it to allow your bridge to run.
```

Reviewer 2: @levihackerman-102, if you add `allow all`, would that work?

Author: Yes @razvand, it works.

Listening on port 8123...
```

> In this step you may face the following error:

Reviewer: ```suggestion
In this step you may face the following error:

```
